### PR TITLE
[NG] Fix datagrid column sizing inside *clrIfActive (#3273)

### PR DESCRIPTION
**Pull request by Jinnie**
Datagrid column sizing lifecycle was calculating column widths
when tab-content was not yet attached to the DOM, resulting in
width: 0px.
Partly reverted the projection back to ng-content.
Added integration test to verify this scenario.

Signed-off-by: Ivan Donchev <idonchev@vmware.com>

### CODE REVIEW

File: src/clr-angular/data/datagrid/datagrid-tabs.integration.spec.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
+/*
```
Comment (hippee-lee) [inline src/clr-angular/data/datagrid/datagrid-tabs.integration.spec.ts]: This test belongs in `datagrid.spec.ts` that's where we do integration testing for datagrid where we need to do more than a pure unit test on a datagrid component. Also, I'm not a fan of adding the `*.integration.spec` filename to the project.

File: src/clr-angular/data/datagrid/datagrid-tabs.integration.spec.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        </clr-tab-content>
+      </clr-tab>
+    </clr-tabs>
+    `,
+})
+class IntegrationTest {
+  items = Array(10).fill(0);
+}
+
+// Verifies issue #3273: https://github.com/vmware/clarity/issues/3273
```
Comment (hippee-lee) [inline src/clr-angular/data/datagrid/datagrid-tabs.integration.spec.ts]: Please remove this its not needed here.

File: src/clr-angular/data/datagrid/all.spec.ts [hunk 3]
```diff
@@ -35,6 +35,7 @@ import DatagridPlaceholderSpecs from './datagrid-placeholder.spec';
 import DatagridRowDetailSpecs from './datagrid-row-detail.spec';
 import DatagridRowSpecs from './datagrid-row.spec';
 import DatagridSpecs from './datagrid.spec';
+import DatagridTabsIntegrationSpecs from './datagrid-tabs.integration.spec';
```
Comment (hippee-lee) [inline src/clr-angular/data/datagrid/all.spec.ts]: This test can be moved into `datagrid.spec.ts`.

Comment (hippee-lee) [inline src/clr-angular/data/datagrid/all.spec.ts]: This test can be moved into `datagrid.spec.ts`.

Comment (hippee-lee) [reply src/clr-angular/data/datagrid/all.spec.ts]: This test can be moved into `datagrid.spec.ts`.

File: src/clr-angular/data/datagrid/datagrid-tabs.integration.spec.ts (hunk 1 above)
Comment (Jinnie) [reply src/clr-angular/data/datagrid/datagrid-tabs.integration.spec.ts]: Moved!

File: src/clr-angular/data/datagrid/datagrid-tabs.integration.spec.ts (hunk 2 above)
Comment (Jinnie) [reply src/clr-angular/data/datagrid/datagrid-tabs.integration.spec.ts]: Removed!

File: src/clr-angular/data/datagrid/all.spec.ts (hunk 3 above)
Comment (Jinnie) [reply src/clr-angular/data/datagrid/all.spec.ts]: Done, done and done :)

**Pull request merged**